components/file_preview_modal/file_preview_modal_header/file_preview_modal_header.test.tsx

```diff
@@ -46,4 +50,44 @@ describe('components/file_preview_modal/file_preview_modal_header/FilePreviewMod
         const wrapper = shallow(<FilePreviewModalHeader {...props}/>);
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should match snapshot with width zoom', () => {
+        const props = {
+            ...defaultProps,
+            toolbarZoom: 'W' as ZoomValue,
+        };
+
+        const wrapper = shallow(<FilePreviewModalHeader {...props}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should match snapshot with height zoom', () => {
+        const props = {
+            ...defaultProps,
+            toolbarZoom: 'H' as ZoomValue,
+        };
+
+        const wrapper = shallow(<FilePreviewModalHeader {...props}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should match snapshot with scale zoom', () => {
+        const props = {
+            ...defaultProps,
+            toolbarZoom: 1.25 as ZoomValue,
+        };
+
+        const wrapper = shallow(<FilePreviewModalHeader {...props}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should match snapshot with arbitrary zoom', () => {
+        const props = {
+            ...defaultProps,
+            toolbarZoom: 1.33 as ZoomValue,
+        };
+
+        const wrapper = shallow(<FilePreviewModalHeader {...props}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
```

Comment:
well done ... you could also add observers to the zoomhandler and emulate a click on the buttons to see if they are called with the right values. see `components/setting_item_max.test.jsx` as an example.

Also wouldn't adding E2E tests here a good option @jgilliam17 ?

Reply:
Yep, I'll do that today

Reply:
I've finally had some time to get to it, sorry about that. They're done now.. Please tell me if there's anything that I should add